apps/meteor/server/startup/migrations/v289.ts

```diff
@@ -0,0 +1,87 @@
+import { LivechatRooms, Messages, OmnichannelServiceLevelAgreements } from '@rocket.chat/models';
+import type { IRocketChatRecord } from '@rocket.chat/core-typings';
+import { LivechatPriorityWeight } from '@rocket.chat/core-typings';
+import type { Db } from 'mongodb';
+
+import { addMigration } from '../../lib/migrations';
+import { isEnterprise } from '../../../ee/app/license/server';
+import { BaseRaw } from '../../models/raw/BaseRaw';
+import { db } from '../../database/utils';
+
+interface IOldLivechatPriority extends IRocketChatRecord {
+	_id: string;
+	name: string;
+	description: string;
+	dueTimeInMinutes: number;
+}
+
+class OldLivechatPriorityRaw extends BaseRaw<IOldLivechatPriority> {
+	constructor(db: Db) {
+		super(db, 'livechat_priority');
+	}
+}
+
+// Migration to migrate old priorities to new SLAs, plus for new priorities feature
+addMigration({
+	version: 289,
+	async up() {
+		const isEE = isEnterprise();
```

Comment:
it is not safe to check this during a migration.. it is not guaranteed a license is applied at this point.. usually licenses coming from Cloud take longer to be applied.

so either remove any check for `isEE` on this migration, or change it to run at some point later.

Reply:
hmm, we copied the idea from the 278 migration 🤔